### PR TITLE
doc: add 2020.1 PU1 setup info for embedded cases

### PR DESCRIPTION
**Pull request by imrickysu**
- Install from package feed on target and sysroot
- Install with bbappend

### CODE REVIEW

File: src/runtime_src/doc/toc/yocto.rst [hunk 1]
```diff
@@ -1,10 +1,90 @@
...
+If the embedded system enabled dnf package manager, user can use dnf tool to install any packages in package feed.
+
+For example, PetaLinux 2020.x package feeds are hosted at http://petalinux.xilinx.com/sswreleases/rel-v2020. 
+User needs to download the package feed specific to the board architecutre and save them to /etc/yum.repos.d directory.
+
+Here's an example to install XRT 2020.1 PU1 for ZCU104.
+
+.. code-block:: bash
+
+        # Excute the following scripts on target embedded system
```
Comment (mamin506) [inline src/runtime_src/doc/toc/yocto.rst:24]: *Execute?

File: src/runtime_src/doc/toc/yocto.rst [hunk 2]
```diff
@@ -1,10 +1,90 @@
...
-Yocto Recipes For Embedded Flow
--------------------------------
+===========================
+XRT Setup for Embedded Flow
+===========================
 
-XRT provide Yocto recipes to build libraries and driver for MPSoC platform.
-This page explains how to build Linux image by PetaLinux Tool.
+To use XRT on embedded designs, xrt and zocl two components need to be installed in the root file system. 
+xrt package contains is user space library and Zocl requires a device tree node.
```
Comment (mamin506) [inline src/runtime_src/doc/toc/yocto.rst:8]: extra "is"?

**Review by mamin506 — APPROVED** [review]

File: src/runtime_src/doc/toc/yocto.rst (hunk 2 above)
Comment (imrickysu) [reply src/runtime_src/doc/toc/yocto.rst:8]: yes. good catch

Comment (imrickysu) [reply src/runtime_src/doc/toc/yocto.rst]: Updates are added.

File: src/runtime_src/doc/toc/yocto.rst (hunk 1 above)
Comment (imrickysu) [reply src/runtime_src/doc/toc/yocto.rst]: Updated

File: src/runtime_src/doc/toc/yocto.rst [hunk 3]
```diff
@@ -1,10 +1,90 @@
...
-Yocto Recipes For Embedded Flow
--------------------------------
+===========================
+XRT Setup for Embedded Flow
+===========================
 
-XRT provide Yocto recipes to build libraries and driver for MPSoC platform.
-This page explains how to build Linux image by PetaLinux Tool.
+To use XRT on embedded designs, xrt and zocl two components need to be installed in the root file system. 
+Xrt package contains user space libraries and zocl is a driver space module, which requires a device tree node.
```
Comment (chvamshi-xilinx) [inline src/runtime_src/doc/toc/yocto.rst:8]: **zocl is a kernel space module.**

File: src/runtime_src/doc/toc/yocto.rst [hunk 4]
```diff
@@ -1,10 +1,90 @@
...
 
-Yocto Recipes For Embedded Flow
--------------------------------
+===========================
+XRT Setup for Embedded Flow
+===========================
 
-XRT provide Yocto recipes to build libraries and driver for MPSoC platform.
-This page explains how to build Linux image by PetaLinux Tool.
+To use XRT on embedded designs, xrt and zocl two components need to be installed in the root file system.
```
Comment (chvamshi-xilinx) [inline src/runtime_src/doc/toc/yocto.rst:7]: xrt and zocl components need to be installed

**Review by chvamshi-xilinx — APPROVED** [review]
Approved, Couple of small changes required..

**Comment by chvamshi-xilinx** [comment]
ok to test

**Pull request merged**